### PR TITLE
{bio}[foss/2017a] RNAcode v0.3

### PR DESCRIPTION
**Pull request by ebirn**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
admin3.novalocal - Linux centos linux 7.4.1708, Intel Xeon E312xx (Sandy Bridge), Python 2.7.5
See https://gist.github.com/aaebf9b64840508eca7a4a2502e5ab01 for a full test report.

**Comment by verdurin** [comment]
@ebirn - welcome, and thanks for the contribution.

There are just some small style aspects that need to be corrected, that I will point out in my review.

File: easybuild/easyconfigs/r/RNAcode/RNAcode-0.3-foss-2017a.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+description = """RNAcode - Analyze the protein coding potential in multiple sequence alignments"""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+
+source_urls = ['http://github.com/downloads/wash/rnacode/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['86c600a07ecd04d243fd6eb3c6cc5b20fd6159f34f69423e7391c0ce8f311048']
+
+dependencies = [
+]
```
Comment (verdurin) [inline easybuild/easyconfigs/r/RNAcode/RNAcode-0.3-foss-2017a.eb]: @ebirn - if there aren't any dependencies, you can simply remove this section completely. Please do so.

File: easybuild/easyconfigs/r/RNAcode/RNAcode-0.3-foss-2017a.eb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+sanity_check_paths = {
+    'dirs': [],
+    'files': ['bin/RNAcode'],
+}
+
```
Comment (verdurin) [inline easybuild/easyconfigs/r/RNAcode/RNAcode-0.3-foss-2017a.eb]: Please remove this unnecessary blank line.

File: easybuild/easyconfigs/r/RNAcode/RNAcode-0.3-foss-2017a.eb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
```
Comment (verdurin) [inline easybuild/easyconfigs/r/RNAcode/RNAcode-0.3-foss-2017a.eb]: @ebirn - please remove this unnecessary blank line.

**Comment by ebirn** [comment]
@verdurin thanks, finally time to give back 👍

**Comment by verdurin** [comment]
Going in, thanks @ebirn!

**Pull request merged**